app/components/content-panel.tsx

```diff
@@ -0,0 +1,28 @@
+import { HomeState } from '@/app/home';
+import getPriceLabel from '@/components/labels';
+import CurrentPrice from '@/components/current-price';
+import { Chart } from '@/components/chart';
+import Image from 'next/image';
+
+export default function ContentPanel({ state }: { state: HomeState }) {
+  return state.zone ? ( // TODO: check more vars. only show if we should
+    <>
+      <CurrentPrice property="Price" label={getPriceLabel(state.priceLevels, state.price)} value={state.price} />
+      <Chart zone={state.zone?.value ?? ''} />
+    </>
+  ) : (
+    // No zone selected view
+    <div className="relative h-[23.5em] w-[100%] max-w-[406px]">
+      <Image
+        src="https://i.pinimg.com/originals/9a/f9/0f/9af90f155c5d30af21494b2afb3e9431.png"
```

Comment:
Perhaps this image should be stored locally instead of accessing a external site which could shutdown/remove content?